pecan/tests/test_util.py

```diff
@@ -93,3 +93,23 @@ def wrapped(*a, **kw):
         actual = util.getargspec(dec(True)(
             self.controller.static_index))
         assert expected == actual
+
+    def test_nested_cells(self):
+
+        def before(handler):
```

Comment:
`handler` is one candidate, but it does not have a `self` (because it's a lambda in the example below)